diab_medications/models.py

```
@@ -0,0 +1,24 @@
+from django.db import models
+import uuid
+
+
+class MedicationOrder(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    patient = models.ForeignKey('patients_core.Patient', on_delete=models.CASCADE)
+    encounter = models.ForeignKey('diab_encounters.Encounter', null=True, blank=True, on_delete=models.SET_NULL)
```

Ruff (0.12.2)

Line too long (112 > 88)

(E501)